packages/@lightningjs/ui-components/src/components/Marquee/Marquee.stories.js

```diff
@@ -69,7 +69,7 @@ Marquee.argTypes = {
     control: 'boolean',
     description: 'Start scrolling on initiation',
     remount: true,
-    table: { defaultValue: { summary: false } }
+    table: { defaultValue: { summary: true } }
```

Comment:
`Marquee.autoStart` by default is `false`. 
[This is where it is where that value is defaulted in the component](https://github.com/rdkcentral/Lightning-UI-Components/blob/develop/packages/%40lightningjs/ui-components/src/components/Marquee/Marquee.js#L79)